benchmarks/docker-k++ADF/Makefile

```
@@ -0,0 +1,6 @@
+
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)